src/test/java/com/hocs/test/pages/complaints/BFProgressCase.java

```diff
@@ -126,7 +150,17 @@ public void completeTheBFStageSoThatCaseMovesToTargetStage(String stageToComplet
                 moveBFCaseFromDraftToQA();
                 break;
             case "QA":
-                moveBCaseFromQAToSend();
+                switch (targetStage.toUpperCase()) {
+                    case "QA":
+                    case "SEND":
+                        moveBCaseFromQAToSend();
```

Comment:
B should be BF

Reply:
haven't made changes to the method call, this was the existing method